.github/ISSUE_TEMPLATE/ui_bug_report_template.yaml

```diff
@@ -0,0 +1,146 @@
+labels: ["bug", "area/uiux"]
+title: "[BUG]"
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for submitting an issue. Please refer to our [issue policy](https://www.github.com/mlflow/mlflow/blob/master/ISSUE_POLICY.md) for additional information about bug reports. For help with debugging your code, please refer to [Stack Overflow](https://stackoverflow.com/questions/tagged/mlflow).
+        #### Please fill in this UI bug report template to ensure a timely and thorough response.
+  - type: dropdown
+    id: contribution
+    attributes:
+      label: Willingness to contribute
+      description: The MLflow Community encourages bug fix contributions. Would you or another member of your organization be willing to contribute a fix for this bug to the MLflow code base?
+      options:
+        - Yes. I can contribute a fix for this bug independently.
+        - Yes. I would be willing to contribute a fix for this bug with guidance from the MLflow community.
+        - No. I cannot contribute a bug fix at this time.
+    validations:
+      required: true
+
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: System information
+      description: |
+        Describe the system where you encountered the bug.
+      value: |
+        - **Have I written custom code (as opposed to using a stock example script provided in MLflow)**:
+        - **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
+        - **MLflow installed from (source or binary)**:
+        - **MLflow version (run ``mlflow --version``)**:
+        - **Python version**:
+        - **yarn version, if running the dev UI**:
+
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Describe the problem
+      description: |
+        Describe the problem clearly here. Include descriptions of the expected behavior and the actual behavior.
+
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Steps to reproduce the bug
+      description: |
+        **Record** steps to reproduce the bug as a video or GIF and share it.
+
+  - type: textarea
+    attributes:
+      label: Code to generate data required to reproduce the bug
+      description: |
+        Please share code to generate data required to reproduce the bug.
+      placeholder: |
+        ```python
+        with mlflow.start_run():
+            mlflow.log_param("p", 0)
+            mlflow.log_metric("m", 1)
+        ```
+
+  - type: textarea
+    attributes:
+      label: Is the console panel showing errors relevant to the bug?
+      description: |
+        If the console panel in the developer tools is showing errors (displayed in red) relevant to the bug, please share them.
+
+        ![console-panel](https://github.com/harupy/mlflow/blob/ui-bug-template/.github/ISSUE_TEMPLATE/images/console-panel.png?raw=true)
```

Comment:
```suggestion
        ![console-panel](https://github.com/mlflow/mlflow/blob/master/.github/ISSUE_TEMPLATE/images/console-panel.png?raw=true)
```

Reply:
I'm referencing images in my fork now for testing. I'll replace them before the merge.